### PR TITLE
fix(material/dialog): Use align as @Input() in MatDialogActions

### PR DESCRIPTION
**Pull request by devversion**
Copy of https://github.com/angular/components/pull/24328. Just to run CI since it didn't run on the author's PRs for quite some time now..

### CODE REVIEW

**Comment by PooSham** [comment]
So the test names must match 1-to-1 between mdc and non-mdc? Can I push a fix directly to your branch, or should I push to my own branch so you can merge?

**Comment by PooSham** [comment]
Or if you want to do it yourself, you can change:

* line 1674 in src/material-experimental/mdc-dialog/dialog.spec.ts 
* line 1737 in src/material/dialog/dialog.spec.ts

to the following line:

    it('should add correct css class according to given [align] input in [mat-dialog-actions]', () => {

File: src/material/dialog/dialog.spec.ts [hunk 1]
```diff
@@ -1733,6 +1733,18 @@ describe('MatDialog', () => {
           .withContext('Expected the aria-labelledby to match the title id.')
           .toBe(title.id);
       }));
+
+      it('should add mat-dialog-actions-align-* class according to given [align] input in [mat-dialog-actions]', () => {
```
Comment (PooSham) [inline src/material/dialog/dialog.spec.ts:1737]: Change to `it('should add correct css class according to given [align] input in [mat-dialog-actions]', () => {`

File: src/material-experimental/mdc-dialog/dialog.spec.ts [hunk 2]
```diff
@@ -1670,6 +1670,17 @@ describe('MDC-based MatDialog', () => {
           .withContext('Expected the aria-labelledby to match the title id.')
           .toBe(title.id);
       }));
+
+      it('should add mat-mdc-dialog-actions-align-* class according to given [align] input in [mat-dialog-actions]', () => {
```
Comment (PooSham) [inline src/material-experimental/mdc-dialog/dialog.spec.ts:1674]: Change to `it('should add correct css class according to given [align] input in [mat-dialog-actions]', () => {`

**Comment by devversion** [comment]
@PooSham you can just push to your PR and I will sync for another CI run.

**Comment by PooSham** [comment]
> @PooSham you can just push to your PR and I will sync for another CI run.

Should be alright now

**Comment by PooSham** [comment]
I think I have fixed the errors now, you can give it another go. @devversion

**Comment by PooSham** [comment]
Woohoo, it went through! What happens now?

**Pull request closed without merging**